admin/server/organizations.go

```diff
@@ -545,9 +547,21 @@ func (s *Server) AddOrganizationMemberUser(ctx context.Context, req *adminv1.Add
 			OrgID:     org.ID,
 			RoleID:    role.ID,
 		})
-		// continue sending an email if the user already exists
-		if err != nil && !errors.Is(err, database.ErrNotUnique) {
-			return nil, status.Error(codes.Internal, err.Error())
+		if err != nil {
```

Comment:
Both approaches would have been fine. But in theory, this approach is slightly better because a) there is no race condition between the check and the insertion/update, b) for the optimistic case where an invite does not yet exist, it only does one DB call instead of two, c) the DB will do a constraint check in any case, so there's no performance issue in relying on it.